lib/MetaCPAN/Web/Controller/Sysadmin/Datacenters.pm

```diff
@@ -0,0 +1,29 @@
+package MetaCPAN::Web::Controller::Sysadmin::Datacenters;
+
+use Moose;
```

Comment:
Have you tried `use MetaCPAN::Moose` here?  Am curious if it causes any problems.